R-package/README.md

```diff
@@ -6,6 +6,9 @@
 ## RGF (Regularized Greedy Forest)
 <br>
 
+**UPDATE 17-08-2018**: Now RGF is part of the [official RGF repository](https://github.com/RGF-team/rgf/tree/master/R-package) and active development is performed there. This repository is archived.
```

Comment:
I think that "This repository is archived." seems to be a little bit confusing here.

Reply:
You are probably right, I changed it to : **The *mlampros/RGF* repository is archived.**

Reply:
Thanks!

Reply:
@StrikerRUS, I've already modified it, you can see it in the [README.md](https://github.com/mlampros/RGF/blob/master/README.md) file. Should I do something more?

Reply:
@mlampros You've modified it in **your repo**, not in this PR. So, after merging this PR, it will seems to be that **this** repo is archived.

Reply:
@StrikerRUS I'm sorry, now I got it.

Reply:
@mlampros No problem! Thank you very much for your work!